_data/package_index.yml

```diff
@@ -382,6 +382,21 @@
 
 # --- Graphics ---
 
+- name: f03gl
+  url: http://www-stone.ch.cam.ac.uk/pub/f03gl/index.xhtml
+  description: Fortran 2003 interface to OpenGL
+  categories: graphics interfaces
+  tags: graphics interface opengl
+  license: GNU GPL v3
+  version: none
```

Comment:
```suggestion
```
We should omit version here for non-github repos.
(This is my fault: the interface isn't consistent between github and non-github repos).

Reply:
Why should we omit it? What's the difference between GitHub and non-GitHub projects with regard to version?

Reply:
Unfortunately I wasn't very consistent when I wrote the Jekyll logic for displaying the version badge. I should change this.

Reply:
@milancurcic 
For github repos:
- omitting the version means use the github-detected version
- `version:none` will hide the version tag

For non-github repos:
- omitting the version will hide the version tag
- `version:none` will create a tag showing 'version:none'